libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -586,7 +586,7 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 			m_context << Instruction::CREATE;
 			// Check if zero (out of stack or not enough balance).
 			m_context << Instruction::DUP1 << Instruction::ISZERO;
-			m_context.appendConditionalInvalid();
+			m_context.appendConditionalRevert();
```

Comment:
@chriseth I'm not completely sure about this case. Is the reasoning here that the caller has no way to prevent this failure and therefore requires that it holds from then on?